src/data/matrix/basic.lean

```diff
@@ -1246,6 +1279,22 @@ begin
   refl
 end
 
+lemma update_row_conj_transpose [decidable_eq m] [has_star α] :
+  update_row Mᴴ j (star c) = (update_column M j c)ᴴ :=
+begin
+  rw [conj_transpose, conj_transpose, transpose_map, transpose_map, update_row_transpose,
+    map_update_column],
```

Comment:
```suggestion
      map_update_column],
```

Reply:
I think we're pretty inconsistent about this, but I'm happy to change it anyway.

Reply:
True, there's no real rule. So I'm happy to keep the indents as-is.